backend/src/controllers/users-controllers.ts

```diff
@@ -4,6 +4,7 @@ import UserModel from "../models/users-models";
 import jwt from "jsonwebtoken";
 
```

Comment:
looks good, but i'll prefer to use jwt in lib/jwt directory in a singleton class